source/clear-linux/get-started/bare-metal-install/bare-metal-manual-install.rst

```diff
@@ -0,0 +1,284 @@
+.. _bare-metal-manual-install:
+
+Bare metal manual installation guide
+####################################
+
+This section is a continuation of steps from the :ref:`bare-metal-install`

```

Comment:
This statement is false. I would change the sentence to something along these lines: This section contains the steps for a |CL| manual installation. It picks up where our :ref:`bare-metal-install` left off:

Reply:
Is the colon at the end of the recommended text required or just a period?